cmd/test-external-check/README.md

```diff
@@ -45,3 +45,46 @@ spec:
           cpu: 10m
           memory: 50Mi
 ```
+
+### image-pull-check
+
+This is a container used to test availability of external image respositories in Kuberhealthy.  Simply place the `kuberhealthy/test-external-check` image on the repository you need tested.
```

Comment:
`This is a container used to test availability of external image respositories in Kuberhealthy.` --> `This container tests the availability of external image repositories using Kuberhealthy`.